src/main/webapp/js/app-config.js

```diff
@@ -27,7 +27,7 @@ define(['angular'], function(angular) {
             'searchURL' : 'https://my.wisc.edu/web/apps/search/'
         })
         .constant('NOTIFICATION', {
-            'enabled' : false,
+            'enabled' : true,
```

Comment:
oh wait, that was from when I was testing notifications way back when. never mind, move along.